src/screens/InsertCardScreen.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+import Prose from '../components/Prose'
+import Main, { MainChild } from '../components/Main'
+import MainNav from '../components/MainNav'
+import Screen from '../components/Screen'
+import Button from '../components/Button'
+
+interface Props {
+  lockScreen: () => void
+}
+
+const InsertCardScreen = ({ lockScreen }: Props) => {
+  return (
+    <Screen>
+      <Main>
+        <MainChild center>
+          <Prose textCenter>
+            <h1>Insert Card</h1>
```

Comment:
suggest `Insert Voter Card`. It's confusing otherwise, poll worker might be wondering which one. Maybe even `Insert Voter Card to Encode`.

Reply:
👍